plugin.sh

```diff
@@ -0,0 +1,9 @@
+CNI_PLUGIN_VERSION="v1.4.0"
+CNI_PLUGIN_TAR="cni-plugins-linux-amd64-$CNI_PLUGIN_VERSION.tgz" # change arch if not on amd64
+CNI_PLUGIN_INSTALL_DIR="/opt/cni/bin"
+
+#https://github.com/containernetworking/plugins/releases/download/v1.4.0/cni-plugins-linux-amd64-v1.4.0.tgz
+curl -LO "https://github.com/containernetworking/plugins/releases/download/$CNI_PLUGIN_VERSION/$CNI_PLUGIN_TAR"
```

Comment:
any reason the infiniband plugin has a dependency on the upstream CNI plugins?

Reply:
It's not needed. Removed.